modules/planning/scenarios/scenario_manager.cc

```diff
@@ -221,8 +221,8 @@ ScenarioConfig::ScenarioType ScenarioManager::SelectTrafficLightScenario(
   const std::vector<PathOverlap>& traffic_light_overlaps =
       reference_line_info.reference_line().map_path().signal_overlaps();
   for (const auto& traffic_light_overlap : traffic_light_overlaps) {
-    const double dist =
-        traffic_light_overlap.start_s - first_encountered_traffic_light.start_s;
+    const double dist = traffic_light_overlap.start_s -
+        first_encountered_traffic_light.start_s;
```

Comment:
Seems like the clang-format tool prefer the existing style.

Reply:
I'll leave it to your bot to update it. 
too much work to keep it unchanged after rebasing...

Reply:
:100: